.npmignore

```diff
@@ -0,0 +1,3 @@
+test
+.github
+package-lock.json
```

Comment:
Let's ditch`package-lock.json` from here (it is automatically ignored by  npm)

Also we have this file just added to repo with #47, so I believe this have to be updated against master

Reply:
Done, Since `History.md` was already there I have added `CHANGELOG.md`